lib/api/controllers/bulkController.js

```diff
@@ -171,6 +172,26 @@ class BulkController extends NativeController {
     return { deleted };
   }
 
+  /**
+   * Directly update every documents matching the search query without:
+   *  - applying max documents write limit
+   *  - injecting Kuzzle metadata
+   *  - triggering realtime notifications
+   */
+  async updateByQuery (request) {
+    const { index, collection } = request.getIndexAndCollection();
+    const query = request.getBodyObject('query');
+    const changes = request.getBodyObject('changes');
+    const refresh = request.getString('refresh', 'false');
```

Comment:
You should use [getRefresh](https://next-docs.kuzzle.io/core/2/framework/classes/kuzzle-request/get-refresh/) instead